pinch/T1B4_GAP.DATA

```diff
@@ -0,0 +1,282 @@
+-- This reservoir simulation deck is made available under the Open Database
+-- License: http://opendatacommons.org/licenses/odbl/1.0/. Any rights in
+-- individual contents of the database are licensed under the Database Contents
+-- License: http://opendatacommons.org/licenses/dbcl/1.0/
+
+-- Copyright (C) 2023 Equinor
+
+-- Test model for MINPV and PINCH. Grid with dimension 1x1x4 with cells in layer 
+-- 2 and 3 inactive due to minpv.   
+
+-- This data deck is identical to T1A except that permz for the inactive layers
+-- are zero. 
+
+-- The model includes 1 well completed in cell 1,1,1. Summary vectors 
+-- BPR (block pressure) will reveal whether there is communication or not 
+-- across inactive cells. 
+
+-- PINCH keyword used with item 2 set to GAP. This will allow communication 
+-- across inactive cells even if gap exceeds threshold specified in item 1. 
+
+-- Actual gap (bottom layer 1 to top layer 4) in this model is 0.5 m and
+-- thereshold is defaulted to 0.001 
+
+-- Item 4 in PINCH keyword is set to ALL which means that pinch transmisibility
+-- is calculated from all tranz including transmissibilities from the inative 
+-- cells.
+
+-- The total gap in the model is larger that the thicknes threshol, but since GAP is 
```

Comment:
Tiny misprint: "thicknes threshol" should probably be "thickness threshold" instead.